portfolio/src/main/webapp/script.js

```diff
@@ -223,18 +223,44 @@ function reset() {
 /**
  * Fetch a greeting from servlet
  */
-function getGreeting() {
+/*function getGreeting() {
   fetch('/data')
   .then(response => response.text())
   .then((greeting) => {
     document.getElementById('greeting-container').innerText = greeting;
   });
+}*/
+
+/**
+ * Fetch comments from servlet
+ */
+function getComments() {
+    fetch('/data')
+    .then(response => response.json())
+    .then((comments)=> {
+        console.log(comments);
+        const commentSection = document.getElementById('comments-container');
+
+        //hard coded for loop to get each of my 3 hard-coded comments
+        for (i = 0; i < 3; i++) {
```

Comment:
Why not use `comments.length` as the bounds?

Reply:
Yeah fair enough, I'll change it. Pretty sure I'm scrapping it next lesson anyway so didn't give it much thought.